src/CLIUtils.ts

```diff
@@ -95,12 +151,24 @@ export async function runFiles(
     );
     Extension = module.default;
   }
+
+  const results: Result[] = [];
   for (const file of files) {
+    const result: Result = {
+      title: '',
+      startedAt: new Date(),
+      finishedAt: new Date(),
+      file,
+      passed: true,
+    };
+
     opts.log && console.log(`Running ${file}...`);
     try {
       const content = readFileSync(file, 'utf-8');
       const object = JSON.parse(content);
       const recording = parse(object);
+      result['title'] = recording.title;
```

Comment:
(nit): let's use dot property accessor like `result.title` (applies to other places as well)